src/main/java/bitwise/DataOutputStream.java

```diff
@@ -27,6 +30,8 @@ public void writeInt(int v) throws IOException {
    * encoding for the character.
    */
   public void writeUTF(String str) throws IOException {
-    // TODO: implement
+    byte[] stringBytes = str.getBytes();
```

Comment:
getBytes: `Encodes this String into a sequence of bytes using the platform's default charset`